apps/projects/app/components/Content/Issues.js

```diff
@@ -251,24 +290,29 @@ class Issues extends React.PureComponent {
                   activeIndex={this.props.activeIndex}
                 />
                 <IssuesScrollView>
-                  {shapeIssues(issuesFiltered).map(issue => (
-                    <Issue
-                      isSelected={this.state.selectedIssues
-                        .map(selectedIssue => selectedIssue.id)
-                        .includes(issue.id)}
-                      onSelect={() => {
-                        this.handleIssueSelection(issue)
-                      }}
-                      onSubmitWork={() => {
-                        this.handleSubmitWork(issue)
-                      }}
-                      onRequestAssignment={() => {
-                        this.handleRequestAssignment(issue)
-                      }}
-                      key={issue.id}
-                      {...issue}
-                    />
-                  ))}
+                  <ScrollWrapper>
+                    {shapeIssues(issuesFiltered).map(issue => (
+                      <Issue
+                        isSelected={this.state.selectedIssues
+                          .map(selectedIssue => selectedIssue.id)
+                          .includes(issue.id)}
+                        onClick={() => {
+                          this.handleIssueClick(issue)
```

Comment:
This action is preventing the other issues actions from occurring. Clicking the buttons still just brings you to the issue detail view.

Reply:
Fixed at cb303b1